apps/docs/package.json

```diff
@@ -24,7 +24,6 @@
     "@fortawesome/react-fontawesome": "^0.2.2",
     "@mdx-js/react": "^1.6.22",
     "@stylexjs/stylex": "0.10.1",
-    "@vercel/analytics": "^1.1.1",
```

Comment:
Fixes incompatible versions of React during docs install

Reply:
Yeah, We can bring this back once I manage to upgrade Docusaurus.